sp/Pedestrian.py

```diff
@@ -306,52 +306,20 @@ def __init__(self, pid, name, start_state, frenet_state, yaw, path, debug_shdata
         self.speed_qualifier = speed_qualifier
         self.reference_speed = reference_speed
         self.scenario_vehicles = scenario_vehicles
+        self.use_collision_point = use_collision_point
```

Comment:
can we eliminate this? If the collision point is provided, we assume it should be used. So this is redundant.